3-reveal/migrations/9-MDA-Lite/deploy/kenya_mda_lite_operational_areas.psql

```diff
@@ -0,0 +1,177 @@
+                    COALESCE (events.form_data->'treated_male_above_15'->>0, '0')::int) AS total_males,
+
+                    SUM(COALESCE (events.form_data->'treated_female_1-4'->>0, '0')::int +
+                    COALESCE (events.form_data->'treated_female_5-14'->>0, '0')::int +
+                    COALESCE (events.form_data->'treated_female_above_15'->>0, '0')::int) AS total_females,
+
+                    SUM(COALESCE (events.form_data->'treated_male_1-4'->>0, '0')::int +
+                    COALESCE (events.form_data->'treated_male_5-14'->>0, '0')::int +
+                    COALESCE (events.form_data->'treated_male_above_15'->>0, '0')::int +
+                    COALESCE (events.form_data->'treated_female_1-4'->>0, '0')::int +
+                    COALESCE (events.form_data->'treated_female_5-14'->>0, '0')::int +
+                    COALESCE (events.form_data->'treated_female_above_15'->>0, '0')::int) AS total_all_genders,
+
+                    SUM(COALESCE (events.form_data->'sum_pzq_received_and_top_up'->>0, '0')::int +
+                    COALESCE (events.form_data->'sum_alb_received_and_top_up'->>0, '0')::int +
+                    COALESCE (events.form_data->'sum_mbz_received_and_top_up'->>0, '0')::int) AS supervisor_distributed,
+
+                    SUM(COALESCE (events.form_data->'received_number'->>0, '0')::int) AS received_number,
+                    SUM(COALESCE (events.form_data->'adminstered'->>0, '0')::int) AS adminstered,
+                    SUM(COALESCE (events.form_data->'damaged'->>0, '0')::int) AS damaged,
+                    SUM(COALESCE (events.form_data->'adverse'->>0, '0')::int) AS adverse,
+
+                    SUM(COALESCE (events.form_data->'received_number'->>0, '0')::int -
+                    (COALESCE (events.form_data->'adminstered'->>0, '0')::int +
+                    COALESCE (events.form_data->'damaged'->>0, '0')::int)) AS remaining_with_cdd,
+
+                    SUM(COALESCE (events.form_data->'pzq_returned'->>0, '0')::int +
+                    COALESCE (events.form_data->'albendazole_returned'->>0, '0')::int +
+                    COALESCE (events.form_data->'mebendazole_returned'->>0, '0')::int) AS returned_to_supervisor
+                FROM events
+                WHERE jurisdictions.structure_jurisdiction_id  = events.location_id
+                AND events.plan_id = jurisdictions.plan_id
+                AND events.entity_type = 'Structure'
+                AND events.event_type IN ('cdd_supervisor_daily_summary', 'tablet_accountability')
+                GROUP BY jurisdictions.plan_id, jurisdictions.structure_jurisdiction_id
+            ) AS subq
+        ) AS daily_summary ON true
+        GROUP BY jurisdictions.plan_id, jurisdictions.structure_jurisdiction_id
+    ) AS operational_area_query ON operational_area_query.plan_id = plans.identifier
+    LEFT JOIN jurisdictions_materialized_view AS kenya_lite_jurisdictions
```

Comment:
Let's replace this with [jurisdictions_tree](https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/3-views/deploy/jurisdictions_tree.psql#L8).  This is because `jurisdictions_materialized_view` is technically deprecated but we cannot yet remove it.

Reply:
Fixed by [this commit](https://github.com/opensrp/opensrp-reveal-datawarehouse/pull/102/commits/bc5a55abe3905abb7807d7fda84e63b0a2a5d6c1).